### DISCUSSION TITLE
Exporting backup to local file system as another option than sharing menu

### DISCUSSION THREAD

**Discussion by krwow** [iSoron/uhabits#2060] [Feature Requests]
Hi. Could an option to make backup locally on device be added? Now after choosing "Export full backup" sharing menu opens and there's no way to save file on device. I prefer such option as I can transfer such file to PC, without using any cloud service. For now I use Shoyo app, but that's just a workaround. It would be great if I didn't have to use additional app just for this one case. I believe it can be done without too much work, as it is usually available in the other apps.

**Comment by krwow** [comment]
@iSoron , any chances to make this happen? Loop Habit Tracker is still the only app I use, which doesn't have saving to a local storage feature.

There are a few ways to achieve that, I assume that from looking at other apps as their user. If not creating an additional option, some of the apps let user choose system "Files" app and then save the backup file using this app.

**Comment by herrbrixter** [comment]
Related to #1912
